### DISCUSSION TITLE
Plugin: Write text

### DISCUSSION THREAD

**Discussion by coolapso** [StreamController/StreamController#37] [Ideas]
Cool to have plugin to write text, currently I am copying to clipboard from streamdeck and pasting with keyboard. 

Due to the x11/wayland transition, maybe decouple both of them and have separate plugins
* Write-text-x11
* Write-text-wayland

**Comment by Core447** [comment]
Added this to the [OS plugin](https://github.com/StreamController/OSPlugin). Most code comes from [streamdeck-linux-gui](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui). It should work on x11 and wayland
